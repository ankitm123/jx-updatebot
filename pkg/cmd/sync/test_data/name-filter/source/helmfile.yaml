filepath: ""
helmfiles:
- path: helmfiles/jx/helmfile.yaml
- path: helmfiles/jx-staging/helmfile.yaml
- path: helmfiles/nginx/helmfile.yaml
templates: {}
renderedvalues: {}
