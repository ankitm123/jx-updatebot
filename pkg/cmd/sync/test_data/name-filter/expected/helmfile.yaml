filepath: ""
helmfiles:
- path: helmfiles/jx-staging/helmfile.yaml
- path: helmfiles/nginx/helmfile.yaml
namespace: jx
templates: {}
renderedvalues: {}
